Ray.jl/test/task.jl

```diff
@@ -1,13 +1,12 @@
 @testset "Submit task" begin
-    oid1 = submit_task(Int32 ∘ length, "hello")
-    addme(x...) = Int32(sum(x))
-    oid2 = submit_task(addme, 1, 2, 3)
+    oid1 = submit_task(length, "hello")
+    oid2 = submit_task(max, 1, 2)
```

Comment:
Slightly more interesting test which results in a non-`Int` return type:
```suggestion
    oid2 = submit_task(max, 0x00, 0xff)
```